src/odata-to-abstract-sql.ts

```diff
@@ -506,7 +508,7 @@ export class OData2AbstractSQL {
 				const unionResource = { ...resource };
 				if (
 					unionResource.definition == null ||
-					!_.isObject(unionResource.definition)
+					typeof unionResource.definition !== 'object'
```

Comment:
Given the typings for this are `Definition | undefined` should we even drop the `typeof` check?

Reply:
I'd leave it as-is unless you want to do extensive testing (as stated elsewhere the typing is what's *intended* but that may not necessarily be followed at all times)